### PR TITLE
Create Calculator component

### PR DESCRIPTION
**Pull request by vvoo21**
### Hi 🙋‍♀️

In this pull request, I created my calculator component and performed the following steps :

-Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called components.
- Inside components, create a new Calculator.js file.
- Once the Calculator is ready, import it and use it in your main component (App.js).

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+          <button type="button" className="btn btn-zero">0</button>
+          <button type="button" className="btn">.</button>
+          <button type="button" className="btn btn-orange ">=</button>
+        </div>
+      </div>
+    </section>
+  );
+}
+
+export default Calculator;
```
Comment (Bondok6) [inline src/components/Calculator.js]: - [ ] Great job here 💯 But according to the requirements, You should use class based components (All components should be class-based). so kindly update this component 👍



- [ ] I really liked your design, It looks great. However, According to the requirements, The `Calculator.js` component should match the following given design ⬇

image

File: src/App.js [hunk 2]
```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```
Comment (Bondok6) [inline src/App.js]: - [ ] The same thing here, kindly check my comment above ⬆
> The App.js component should be class-based.

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @vvoo21 🙋‍♂️ , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No Linter errors ✅
- Correct GitHub flow ✅
- Your repo is professional ✅
- Professional README file ✅


## Required Changes ♻️
- [ ] _Check the comments under the review._ ⬇

## Optional suggestions

- [OPTIONAL] Your README is well documented 💯 However, I noticed that the MIT Link in your README is broken, so I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

- [OPTIONAL] Also, I suggest removing this `NOTE ...` recommendation ( Last Line ) from your README file 👍


> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- 

### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**